bindings/stronghold-nodejs/README.md

```diff
@@ -13,6 +13,49 @@ Development Release: this version matches the dev branch of this repository, may
 ```bash
 npm install @iota/identity-stronghold-nodejs@dev
 ```
+## Usage
+<!-- 
+Test this example using https://github.com/anko/txm: `txm README.md`
+
+Replace imports with local paths for txm:
+!test program
+cat \
+| sed -e "s#require('@iota/identity-wasm/node')#require('../wasm/node/identity_wasm.js')#" \
+| sed -e "s#require('@iota/identity-stronghold-nodejs')#require('./dist/index.js')#" \
+| node
+-->
+<!-- !test check Nodejs Example -->
+```javascript
+const { AccountBuilder, ExplorerUrl } = require('@iota/identity-wasm/node')
+const { Stronghold } = require('@iota/identity-stronghold-nodejs')
+
+
+async function main() {
+    
+    const stronghold = await Stronghold.build("./example-strong.hodl", "my-password", true);
```

Comment:
We should a bit more explanation for these parameters, similar to the root README.md. I think we can omit the `dropsave` parameter since it's slightly more advanced/distracting and defaults to `true` anyway.

```suggestion
    // Stronghold settings for the Account storage.
    // This will load an existing Stronghold or create a new one automatically.
    const filepath = "./example-strong.hodl";
    const password = "my-password";
    const stronghold = await Stronghold.build(filepath, password);
```